LICENSE.md

```diff
@@ -0,0 +1,7 @@
+## Copyright 2023, [Harrison Njuguna]
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this [Blog App] and associated documentation files, to deal in the [Blog App] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [Blog App], and to permit persons to whom the [Blog App] is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the [Blog App].
+
+THE [Blog App] IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE [Blog App] OR THE USE OR OTHER DEALINGS IN THE [Blog App].
```

Comment:
- [ ] Kindly fix the link to the license for this project to be approved. Currently, I am still getting a 404 error.

![licence](https://user-images.githubusercontent.com/53124548/216751396-939b8ff5-335b-4742-879d-cad28d1f9a59.png)